readinglist/authentication.py

```diff
@@ -79,7 +78,7 @@ def _get_credentials(self, request):
             raise HTTPServiceUnavailable()
         except (fxa_errors.InProtocolError, fxa_errors.TrustError):
             return None
-        return user_id
+        return 'fxa_%s' % user_id
```

Comment:
Even if it is not a big deal, this will deserve a note in the changelog : all records lost ;)